.github/workflows/deploy-feature-envs.yaml

```diff
@@ -450,7 +551,10 @@ jobs:
           environment: ${{ steps.delete_fe.outputs.deployment-name }}
 
   destroy_feature_env_on_close:
-    if: (github.event.action == 'closed' && !contains( github.event.pull_request.labels.*.name, 'DESTROYED') && contains( github.event.pull_request.labels.*.name, 'DEPLOYED'))
+    if: >
```

Comment:
Let's re-format that `if` statement so it's actually transparent.  There are few brackets and it's tricky to get the logic behind it.  We could try something like:
````
if: >
  github.event.action == 'closed'
  && !contains(github.event.pull_request.labels.*.name, 'DESTROYED')
  && contains(github.event.pull_request.labels.*.name, 'DEPLOYED')
````
Do we need brackets on the outside?  I have removed them in this comment

Reply:
done